docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+services:
+  todo-service-database:
+    image: mongo:latest
+    container_name: "todo-service-database"
+    expose:
+      - 27017
+    ports:
+      - "27017:27017"
+    volumes:
+      - todo:/var/lib/mongodb
+  todo-service-api:
+    image: todo-service-image
+    container_name: todo-service-api
+    expose:
+      - 8081
+    ports:
+      - "8081:8080"
+    restart: always
+    links:
```

Comment:
it better to use `depends_on` it means that your `todo-service-api` would wait until db will be run

Reply:
links → depends_on